techlibs/ice40/ice40_dsp.pmg

```diff
@@ -46,17 +46,24 @@ code sigA sigB sigH
 
 	// Only care about those bits that are used
 	int i;
-	for (i = 0; i < GetSize(O); i++) {
-		if (nusers(O[i]) <= 1)
-			break;
-		sigH.append(O[i]);
-	}
+	for (i = GetSize(O) - 1; i > 0 && nusers(O[i]) <= 1; i--)
+		;
 	// This sigM could have no users if downstream sinks (e.g. $add) is
 	//   narrower than $mul result, for example
 	if (i == 0)
```

Comment:
Is this bit still correct?

Reply:
also this makes for a good opportunity to figure out what the comment referencing a non-existent `sigM` is actually supposed to say